specification/devcenter/DevCenter/Environments/models.tsp

```diff
@@ -126,15 +194,128 @@ Properties of an environment. These properties can be updated after the
 resource has been created.
 """)
 model EnvironmentUpdateProperties {
+  ...EnvironmentPatchProperties;
+
   #suppress "@azure-tools/typespec-azure-core/bad-record-type" "there is no build in type to describe object"
   @doc("Parameters object for the environment.")
   parameters?: Record<unknown>;
 }
 
+@doc("""
+Properties of an environment. These properties can be updated via PATCH after
+the resource has been created.
+""")
+@added(APIVersions.v2024_02_01)
+model EnvironmentPatchProperties {
```

Comment:
I understand, but I also think this is ok that way. This is more of a visual way to distinguish which properties are going to be updated using "environemtCreateOrUpdate",  and the ones that are going to be updated with "environmentPatch". But there is no actual value in the `EnvironmentUpdateProperties` and `EnvironmentPatchProperties` models - if you check they don't even appear in the swagger as models, neither sdk models are generated from it. Those were the original names given in the swagger and i think it's ok keeping them

Reply:
Yeah it's fine if you want to keep them separated out and with their own name in the specification, but let's make this an alias since that's the most correct thing for typespec in this case.

Reply:
This conversation made me realize that EnvironmentUpdateProperties should also have been an alias :/ I'll check on what we should do for that with an architect.

Reply:
Since we added omit unreachable properties in tspconfig.yaml seems like we could also update EnvironmentUpdateProperties to an alias IF the stable SDKs dont reference it.